selfdrive/ui/qt/onroad.h

```diff
@@ -17,19 +17,19 @@ class OnroadAlerts : public QWidget {
   Q_OBJECT
 
 public:
-  OnroadAlerts(QWidget *parent = 0) {};
+  OnroadAlerts(QWidget *parent = 0) : QWidget(parent) {};
+  QColor updateState(const UIState &s);
```

Comment:
updateState should be a standard slot all UI widgets have, don't like that this one returns a QColor. we can add a getter for the color or just abstract out the logic to get the background color from the alert status

Reply:
Yes,that's better